Wastewater_Seq_analysis/WWP_seq_initialize_analysis.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# if any errors, then exit
+set -euxo pipefail 
+
+###########################
+# Author: Pooja Gupta
+
+USAGE="
+Purpose: 
+1) Copy fastq files from CovidSeq analysis directory to wastewater sequencing analysis directory on NAS.
+2) Setup directory structure for initiating Wastewater sequencing run analysis and any downstream analysis.
+3) Generate ncbi submission folder that can be directly used for uploading files to NCBI.
+
+Usage:
+sh WWP_seq_initialize_analysis.sh <wastewater sequencing run_name> | tee -a WWP_seq_initialize_analysis.log
+Last updated on June 16,2022
+"
+###########################
+
+echo $USAGE
+
+run_name=$1
+log_files=/Volumes/NGS/Analysis/covidseq/$run_name/covidseq_output/Logs_Intermediates
+
+if [ ! -d "$log_files/VirusDetection" ]
+then
+    echo "$(date) : Fastq generation step is not yet completed for run $run_name" 
+    exit 1
+fi
+
+echo "$(date) : Fastq generation step is completed for run $run_name."
+
+sample_sheet=$(ls $log_files/SampleSheetValidation/SampleSheet_Intermediate.csv | head -n 1)
+if [ ! -n "$sample_sheet" ]
+then
+    echo "$(date) : The sample sheet for run $run_name does not exist. Expected $sample_sheet"
+    exit 1
+fi
+echo "$(date) : The sample sheet for run $run_name is $sample_sheet"
+
+#Check whether sample sheet contains any wastewater samples. This will mostly be true.            
+if grep -nqi 'Wastewater' $sample_sheet
+then
+    echo "$(date) : Some wastewater samples found in $run_name"
+    else 
+        echo "$(date) : No wastewater samples found in $run_name .. exiting"
+        exit 1
```

Comment:
exit 1 will leave an error message. I would recommend exit 0 if this is a neutral exit condition.